### PR TITLE
Fixed background color for embeds + generic ->color hash function

### PR DESCRIPTION
**Pull request by rafd**

### CODE REVIEW

File: src/braid/ui/views/embed.cljs [hunk 1]
```diff
@@ -11,7 +12,7 @@
 
 (defn- website-embed-view [content]
   [:div.content.loaded.website
-   {:style {:background-color (str "hsl(" (* 360 (rand)) ",70%,35%)")}}
+   {:style {:background-color (->color (content :provider_name))}}
```
Comment (jamesnvc) [inline src/braid/ui/views/embed.cljs]: provider_name vs url? I guess this why things from the same site have the same colour?

Comment (rafd) [inline src/braid/ui/views/embed.cljs]: Indeed, I figured it'd be a nice little bonus.

File: src/chat/client/views/helpers.cljs [hunk 2]
```diff
@@ -6,23 +6,22 @@
...
-                       str
-                       (.substring 33 36)
-                       (js/parseInt 16)
-                       (/ 4096))]
-    (str "hsl(" (* 360 normalized) ",70%,35%)")))
+(defn ->color [input]
+  (str "hsl(" (mod (Math/abs (hash input)) 360) ",71%,35%)"))
+
+(defn id->color [uuid]
+  (->color uuid))
```
Comment (jamesnvc) [inline src/chat/client/views/helpers.cljs]: Everything has changed colour now! :sob: 
Also, better to just replace uses of `id->color` with `->color`?

Comment (rafd) [inline src/chat/client/views/helpers.cljs]: Yes, we should use `->color` but that can be a seperate PR

**Pull request merged**